contracts/DCAStrategies/DCAStrategies/DCAStrategiesPositionsHandler.sol

```diff
@@ -371,6 +372,9 @@ abstract contract DCAStrategiesPositionsHandler is IDCAStrategiesPositionsHandle
             positionId: _currentPositionId,
             amount: _correspondingToPosition - _userPosition.remaining
           });
+        } else {
+          // do nothing
+          _tasks[_data.newPositionsIndex] = Task({action: Action.NOTHING, positionId: _currentPositionId, amount: 0});
```

Comment:
Is this necessary? Since `NOTHING` is the first value in the enum, it will be the default value, so I think it might not be necessary to assign it

Reply:
I think it is. I need it to save the positionId to be assigned in the last for loop. I can reuse existing enum values if you want. (In line 377 when writing the task I could use the REDUCE value or any other. It doesn't matter because is not read).